pilot/lib/pilot/commands_generator.rb

```diff
@@ -158,11 +158,5 @@ def run
 
       run!
     end
-
-    def create_config(options)
```

Comment:
I actually like having this method here, I think it's cleaner than having `FastlaneCore::Configuration.create(Pilot::Options.available_options, convert_options(options))` sprinkled throughout. 
It pushes off the implementation details of creating a config to a single method and IMO looks better at the call site.
Maybe just change the method so that it only does `FastlaneCore::Configuration.create` work

This is of course just an opinion, so take it our leave 👍